src/footprints/TRRS-PJ-320A-dual.js

```diff
@@ -0,0 +1,55 @@
+module.exports = {
+  nets: ['A', 'B', 'C', 'D'],
+    params: {
+        class: 'T', // for TRRS
+		reverse: true

```

Comment:
First off, please use space-based indentation like the rest of the files (the tabs tend to mess things up, especially on GH).